packages/fxa-auth-server/fxa-oauth-server/lib/error.js

```diff
@@ -295,4 +295,20 @@ AppError.invalidGrantType = function invalidGrantType() {
   });
 };
 
+AppError.serviceUnavailable = function serviceUnavailable(retryAfter) {
+  if (! retryAfter) {
+    retryAfter = 30;
+  }
+  return new AppError({
+    code: 503,
+    error: 'Service Unavailable',
+    errno: 201,
```

Comment:
We'll have to add this to the [list of front-end OAuth server errors](https://github.com/mozilla/fxa/blob/dc06472fd5278f179065a75779fdfe4baa4efa39/packages/fxa-content-server/app/scripts/lib/oauth-errors.js#L100).